.github/workflows/rubocop.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         internal_investigation: [null]
         include:
           - { rubocop: master, ruby: "3.0", os: ubuntu }
-          - { rubocop: "v1.61.0", ruby: "3.3", os: ubuntu }
+          - { rubocop: "v1.69.2", ruby: "3.4", os: ubuntu }
```

Comment:
I guess, this is supposed to test that rubocop-ast still works with older rubocop? Would it then make more sense to instead test master against the latest ruby version so that this can be left with and older version?

Reply:
I don't have strong opinions about this, this PR aligns with the previous upgrade:
https://github.com/rubocop/rubocop-ast/pull/274

Reply:
I can open a followup